components/logins/src/store.rs

```diff
@@ -151,6 +165,16 @@ impl LoginStore {
         Ok(())
     }
 
+    #[handle_error(Error)]
+    pub fn set_last_sync(self: Arc<Self>, last_sync: ServerTimestamp) -> ApiResult<()> {
```

Comment:
is this needed now `verify_logins` calls it?

Reply:
I left a comment about this below but this function only exists for the `test_login_local_delete` integration test. I'll delete both of these functions now as I created that test to validate my overall approach.